### ISSUE TITLE
Add a project tag in sliderule-base.pkr.hcl to capture costs for the prov-sys

### ISSUE THREAD

**Issue by cugarteblair** [SlideRuleEarth/sliderule#297]
inside the default_tags. tags= section
Project = "cluster-${var.cluster_name}"

`locals {
 terraform-git-repo = "sliderule-cluster"
}

provider "aws" {
  region  = var.aws_region
  default_tags {
    tags = {
      Owner   = "SlideRule"
      Project = "cluster-${var.cluster_name}"
      terraform-base-path = replace(path.cwd,
      "/^.*?(${local.terraform-git-repo}\\/)/", "$1")
      cost-grouping = "${var.cluster_name}"
    }
  }
}`

**Comment by jpswinski** [comment]
@cugarteblair - I went to add the project tag to the packer file, and I realized that there is no concept of a cluster for the packer build.  We use the same AMI across all cluster and all types of instances.  I think we want to keep it that way because it greatly simplifies managing it.  Also, for security reasons, we never want to go back to an old AMI, or use anything other than the latest.

**Comment by cugarteblair** [comment]
JP,It seems to me that the AMI and project tag are independent issues. Correct?Carlos CarlosOn Aug 22, 2023, at 7:50 AM, JP Swinski ***@***.***> wrote:﻿
@cugarteblair - I went to add the project tag to the packer file, and I realized that there is no concept of a cluster for the packer build.  We use the same AMI across all cluster and all types of instances.  I think we want to keep it that way because it greatly simplifies managing it.  Also, for security reasons, we never want to go back to an old AMI, or use anything other than the latest.

**Comment by cugarteblair** [comment]
Is the issue that the packer is built independent of instantiating the EC2?

**Comment by jpswinski** [comment]
Yes, that's right.  We maintain only three AMI's: "v3", "latest", and "unstable".  In practice, v3 and latest are the only ones I keep up-to-date.  We don't keep fully specified versions or old versions, because as soon as an AMI needs to be updated, it has to be updated across the board for every cluster.
